kernel/safe_typing.mli

```diff
@@ -35,7 +35,10 @@ val is_initial : safe_environment -> bool
 
 val env_of_safe_env : safe_environment -> Environ.env
 
-val sections_of_safe_env : safe_environment -> section_data Section.t
+val sections_of_safe_env : safe_environment -> section_data Section.t option
+
+val force_sections_of_safe_env : safe_environment -> section_data Section.t
```

Comment:
Why export this if there are no users? I don't think it's the job of the kernel to display nice messages when there is a programming error.

Reply:
There's a user in lib

Reply:
Ah, yes, sorry. (Still seems strange to export, but fine.)

Reply:
We could move it to lib and make it internal

Reply:
I'd prefer so if possible. Not to mention the fact that this is quite ill-named...

Reply:
Changed